tools/check-license-header.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to SkyAPM org under one or more contributor
```

Comment:
This file looks familiar 🧐

Reply:
https://github.com/SkyAPM/go2sky/blob/master/tools/check-license-header.py Here is also :)

Reply:
and this https://github.com/apache/skywalking-python/blob/master/tools/check-license-header.py 🤪